### PR TITLE
HSC-124: Add Missing Drug Concepts

### PR DESCRIPTION
**Pull request by nuhuhmutebi**

### CODE REVIEW

File: configuration/concepts/drugs_concepts.csv [hunk 1]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
 bb9c9e33-00db-4f9f-b230-adde3df9b092,,,Primoquine,Primoquine,Drug,N/A,,,
 497457dc-48c1-40f9-8261-0bd9f4bf6b21,,,Dyphenhydramine,Dyphenhydramine,Drug,N/A,,,
 48bcdab5-adb0-4229-aef3-1168a51c2932,,,Eucalyptus & Diphenhydramine & Phenylephrine,Eucalyptus & Diphenhydramine & Phenylephrine,Drug,N/A,,,
-984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
\ No newline at end of file
+984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
+a546446e-e789-4d02-b02f-c938c6bd373f,,,Ophtalnol,ophtalnol,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us change this to 'Boric acid & Antipyrine & Ephedrine' and 'Acide borique & Antipyrine & Ephedrine' for en and fr respectively

File: configuration/concepts/drugs_concepts.csv [hunk 2]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
 bb9c9e33-00db-4f9f-b230-adde3df9b092,,,Primoquine,Primoquine,Drug,N/A,,,
 497457dc-48c1-40f9-8261-0bd9f4bf6b21,,,Dyphenhydramine,Dyphenhydramine,Drug,N/A,,,
 48bcdab5-adb0-4229-aef3-1168a51c2932,,,Eucalyptus & Diphenhydramine & Phenylephrine,Eucalyptus & Diphenhydramine & Phenylephrine,Drug,N/A,,,
-984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
\ No newline at end of file
+984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
+a546446e-e789-4d02-b02f-c938c6bd373f,,,Ophtalnol,ophtalnol,Drug,N/A,,,
+252159dc-3a8c-4587-801f-762118de992c,,,Belladone & phenobarbital,Belladone & phenobarbital,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us change the en to 'Belladona & Phenobarbital'

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/drugs_concepts.csv [hunk 3]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
 bb9c9e33-00db-4f9f-b230-adde3df9b092,,,Primoquine,Primoquine,Drug,N/A,,,
 497457dc-48c1-40f9-8261-0bd9f4bf6b21,,,Dyphenhydramine,Dyphenhydramine,Drug,N/A,,,
 48bcdab5-adb0-4229-aef3-1168a51c2932,,,Eucalyptus & Diphenhydramine & Phenylephrine,Eucalyptus & Diphenhydramine & Phenylephrine,Drug,N/A,,,
-984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
\ No newline at end of file
+984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
+a546446e-e789-4d02-b02f-c938c6bd373f,,,Ophtalnol,ophtalnol,Drug,N/A,,,
+252159dc-3a8c-4587-801f-762118de992c,,,Belladone & phenobarbital,Belladone & phenobarbital,Drug,N/A,,,
+8f8920be-f144-4fba-b4db-d32940dcea04,,,Desqualinium & lidocaine,Desqualinium & lidocaïne,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Lets change to 'Dequalinium & Lidocaine' for en

File: configuration/concepts/drugs_concepts.csv [hunk 4]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
 bb9c9e33-00db-4f9f-b230-adde3df9b092,,,Primoquine,Primoquine,Drug,N/A,,,
 497457dc-48c1-40f9-8261-0bd9f4bf6b21,,,Dyphenhydramine,Dyphenhydramine,Drug,N/A,,,
 48bcdab5-adb0-4229-aef3-1168a51c2932,,,Eucalyptus & Diphenhydramine & Phenylephrine,Eucalyptus & Diphenhydramine & Phenylephrine,Drug,N/A,,,
-984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
\ No newline at end of file
+984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
+a546446e-e789-4d02-b02f-c938c6bd373f,,,Ophtalnol,ophtalnol,Drug,N/A,,,
+252159dc-3a8c-4587-801f-762118de992c,,,Belladone & phenobarbital,Belladone & phenobarbital,Drug,N/A,,,
+8f8920be-f144-4fba-b4db-d32940dcea04,,,Desqualinium & lidocaine,Desqualinium & lidocaïne,Drug,N/A,,,
+05130695-8005-4d57-a803-111265f0f4c1,,,Diclofenac & B complex,Diclofénac & complexe B,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us change to 'Diclofenac & Vitamin B complex' and 'Diclofénac & Complexe de vitamine B'

File: configuration/concepts/drugs_concepts.csv [hunk 5]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
...
 497457dc-48c1-40f9-8261-0bd9f4bf6b21,,,Dyphenhydramine,Dyphenhydramine,Drug,N/A,,,
 48bcdab5-adb0-4229-aef3-1168a51c2932,,,Eucalyptus & Diphenhydramine & Phenylephrine,Eucalyptus & Diphenhydramine & Phenylephrine,Drug,N/A,,,
-984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
\ No newline at end of file
+984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
+a546446e-e789-4d02-b02f-c938c6bd373f,,,Ophtalnol,ophtalnol,Drug,N/A,,,
+252159dc-3a8c-4587-801f-762118de992c,,,Belladone & phenobarbital,Belladone & phenobarbital,Drug,N/A,,,
+8f8920be-f144-4fba-b4db-d32940dcea04,,,Desqualinium & lidocaine,Desqualinium & lidocaïne,Drug,N/A,,,
+05130695-8005-4d57-a803-111265f0f4c1,,,Diclofenac & B complex,Diclofénac & complexe B,Drug,N/A,,,
+66c997ab-d252-43e5-94ca-b5925a60fdbb,,,Diclofenac & Methocarbamol & paracetamol,Diclofenac & Methocarbamol & paracetamol,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: 'Diclofenac & Methocarbamol & Paracetamol' for both en and fr

File: configuration/concepts/drugs_concepts.csv [hunk 6]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
...
+5ac2861d-71e9-4488-bf29-449d555ddfcb,,,Diosmine & Hesperidine,Diosmine & Hesperidine,Drug,N/A,,,
+909ea570-031b-422e-b28d-17b7f492a1b9,,,Genfibrozil,Gemfibrozil,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: 'Gemfibrozil' for en and 'Genfibrozil' for fr

File: configuration/concepts/drugs_concepts.csv [hunk 7]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
...
+1d47e474-4ae6-4e47-9b69-2ce6ed7399ac,,,Glucosamine & chondroitin,Glucosamine chondroïtine,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: 'Glucosamine & Chondroitin' for both

File: configuration/concepts/drugs_concepts.csv [hunk 8]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
...
+a30dc1f2-a26b-4bc0-90a4-e7eb7feea473,,,Nystatin & Zinc Oxyde,Nystatine et oxyde de zinc,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: lets change to 'Nystatin & Zinc Oxide' for english

File: configuration/concepts/drugs_concepts.csv [hunk 9]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
...
+bcf73538-3f39-4429-a30d-8499609942cd,,,Multivitamin & Iron,Multivitamines et fer,Drug,N/A,,,
+d644e9bd-7ae2-4ef3-b7b7-739816051213,,,Pentoxyphillin,Pentoxyphilline,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let's change the en to 'Pentoxifylline' for both en and fr

File: configuration/concepts/drugs_concepts.csv [hunk 10]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
...
+792871a0-4a52-4e5d-bdd2-4fd49d636227,,,Podophillin,Podophylline,Drug,N/A,,,
+f72393c8-fda0-4107-9b4b-017888be72b2,,,Vitamin A & D, Vitamine A et D,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let's remove this and keep using [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L463)

File: configuration/concepts/drugs_concepts.csv [hunk 11]
```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
...
+916e4268-45b9-4db4-b273-1d5b4a6539b3,,,Bismuth Sub-Salicylate,Sous-Salicylate de bismuth,Drug,N/A,,,
```
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let's change the en to 'Bismuth subsalicylate'

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/drugs_concepts.csv (hunk 11 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: None

File: configuration/concepts/drugs_concepts.csv (hunk 10 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 9 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 8 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 7 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 6 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 5 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 4 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 3 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 2 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 1 above)
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

**Pull request merged**